src/qt/peertablesortproxy.cpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2020 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <qt/peertablesortproxy.h>
+
+#include <qt/peertablemodel.h>
+#include <util/check.h>
+
+#include <QModelIndex>
+#include <QString>
+#include <QVariant>
+
+PeerTableSortProxy::PeerTableSortProxy(QObject* parent)
+    : QSortFilterProxyModel(parent)
```

Comment:
nit: since you use `{}` in `m_peer_table_sort_proxy{nullptr};`, maybe worth using `{}` in `QSortFilterProxyModel{parent}` too (meaning in the new code)?

Reply:
As stated in the commit description, the goal is to follow Qt [docs](https://doc.qt.io/qt-5/qabstracttablemodel.html#index).

Reply:
This commit belongs to the #18 on which this one is based.